### PR TITLE
Accessibility checks

### PR DESCRIPTION
**Pull request by dazoakley**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -219,6 +222,42 @@ jobs:
       - store_artifacts:
           path: integration_tests/screenshots
 
+  accessibility_test:
+    executor: integration-tests
+    steps:
+      - browser-tools/install-chrome
```
Comment (jonnywyatt) [inline .circleci/config.yml:228]: do we definitely need these? The existing integration_test job doesn't install them

File: integration_tests/support/commands.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { exactMatchIgnoreWhitespace, splitIsoDateToParts } from './utils'
 import { recall } from '../fixtures/recall'
 import { getPrisonerResponse, getRecallResponse } from '../mockApis/mockResponses'
+import 'cypress-file-upload'
```
Comment (jonnywyatt) [inline integration_tests/support/commands.js:4]: This line got moved to support/index.js

File: package.json [hunk 3]
```diff
@@ -22,15 +22,17 @@
     "test": "jest --forceExit",
     "test:coverage": "jest --forceExit --coverage",
     "security_audit": "npx audit-ci --config audit-ci.json --skip-dev",
-    "int-test": "rm -rf cypress/downloads && cypress run",
-    "int-test-ui": "rm -rf cypress/downloads && cypress open",
+    "int-test": "rm -rf cypress/downloads && cypress run --browser chrome",
+    "int-test-ui": "rm -rf cypress/downloads && cypress open --browser chrome",
```
Comment (jonnywyatt) [inline package.json:26]: It's not doing any harm being there, but the `--browser` switch won't have any effect locally as chrome is always the default anyway with `cypress open`

File: scripts/start-for-integration-tests.sh [hunk 4]
```diff
@@ -31,11 +31,15 @@ echo "  docker logs fake-manage-recalls-api"
 echo "  docker logs redis"
 
 echo "Starting ${MANAGE_RECALLS_UI_NAME}, logs can be found here: ${LOG_FILE}"
-npm run start:feature >> "${LOG_FILE}" 2>&1 &
+npm run start:feature >>"${LOG_FILE}" 2>&1 &
```
Comment (jonnywyatt) [inline scripts/start-for-integration-tests.sh:34]: Do we need the `sleep` ? It worked without it

File: integration_tests/accessibility/accessibility.spec.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+    cy.task('expectListRecalls', {
+      expectedResults: [
+        {
+          ...basicRecall,
+          status: 'BEING_BOOKED_ON',
+        },
+      ],
+    })
+    cy.task('expectGetRecall', {
```
Comment (jonnywyatt) [inline integration_tests/accessibility/accessibility.spec.js:52]: We can share these mocks with the main integration tests, later on

File: integration_tests/accessibility/accessibility.spec.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
+import { getRecallResponse, getAllFieldsHistoryResponseJson } from '../mockApis/mockResponses'
+
+const pa11yArgs = { runners: ['axe'], standard: 'WCAG2AA' }
+const nomsNumber = 'A1234AA'
+const recallId = '123'
+const urls = [
+  '/',
```
Comment (jonnywyatt) [inline integration_tests/accessibility/accessibility.spec.js:8]: maybe worth adding:
- confirmation page (for either book, assess or dossier)
- user details
- check dossier details (ie first page after you click Create dossier)
- change history (documents tab)
- change history for a document
- last known address page

File: .circleci/config.yml (hunk 1 above)
Comment (dazoakley) [reply .circleci/config.yml:228]: I switched everything to run on chrome (integration and accessibility) as electron was masking some of the a11y issues.  Can change the integration tests back to electron if you like though?

**Comment by jonnywyatt** [comment]
I think upgrading NPM version might be better in a separate PR, as it could cause large changes to the package-lock.json that we'd want to isolate

File: integration_tests/support/commands.js (hunk 2 above)
Comment (dazoakley) [reply integration_tests/support/commands.js:4]: Ahh, I did wonder about that - came through in the rebase this morning - will remove

File: scripts/start-for-integration-tests.sh (hunk 4 above)
Comment (dazoakley) [reply scripts/start-for-integration-tests.sh:34]: Never did on my machine 😢

File: integration_tests/accessibility/accessibility.spec.js (hunk 5 above)
Comment (dazoakley) [reply integration_tests/accessibility/accessibility.spec.js:52]: Yarp 👍

File: integration_tests/accessibility/accessibility.spec.js (hunk 6 above)
Comment (dazoakley) [reply integration_tests/accessibility/accessibility.spec.js:8]: We've added all the above except the change history (for documents) ones as we couldn't figure out how to get the mocks to work... Maybe add this in then pair up to get those (and checking error states) in?

**Comment by dazoakley** [comment]
> I think upgrading NPM version might be better in a separate PR, as it could cause large changes to the package-lock.json that we'd want to isolate

Every time I or Shahnaaz run the code we end up with massive `package-lock.json` changes... But yep, can roll this one out.

File: .circleci/config.yml (hunk 1 above)
Comment (jonnywyatt) [reply .circleci/config.yml:228]: ah! I see sorry - good shout

**Pull request merged**